src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -406,8 +406,18 @@ public function getVersions()
     public function getVersionLog()
     {
         $result = array();
-        $rows = $this->fetchAll(sprintf('SELECT * FROM %s ORDER BY version ASC', $this->getSchemaTableName()));
-        foreach ($rows as $version) {
+
+        switch ($this->options['version_order']) {
+            case \Phinx\Config\Config::VERSION_ORDER_CREATION_TIME:
```

Comment:
Internally, I think it would be nicer to see `VERSION_ORDER_VERSION = 'version';` / `VERSION_ORDER_START_TIME = 'start_time` as these are the names of the columns in the phinxlog table.

Externally (in the config), the end user setup may be simpler to have `version_order` be one of `'execution'` or `'creation'`.